packages/charts/src/chart_types/heatmap/state/selectors/get_grid_full_height.ts

```diff
@@ -31,21 +33,26 @@ export const getGridHeightParamsSelector = createCustomCachedSelector(
     getParentDimension,
     getHeatmapConfigSelector,
     getHeatmapTableSelector,
+    getHeatmapSpecSelector,
+    getChartThemeSelector,
   ],
   (
     legendSize,
     { showLegend },
     { height: containerHeight },
     { xAxisLabel: { padding, visible, fontSize }, grid, maxLegendHeight },
     { yValues },
+    { xAxisTitle },
+    { axes: axesStyle },
   ): GridHeightParams => {
     const xAxisHeight = visible ? fontSize : 0;
     const totalVerticalPadding = padding * 2;
```

Comment:
I think we get rid of this `* 2` logic.

```suggestion
    const totalVerticalPadding = padding; // or just replace totalVerticalPadding with padding below
```

Reply:
Removed in eb9c5ab thanks!